src/controllers/ArticlesController.js

```diff
@@ -0,0 +1,92 @@
+import status from '../config/status';
+import { generateReadTime, generateSlug } from '../helpers';
+
+import { Article } from '../queries';
+
+/**
+ * Author: Gilles Kagarama
+ * @returns {object} Object representing the response returned
+ */
+class ArticleController {
```

Comment:
As I see here, the file is ``ArticlesController.js`` but the class name is ``ArticleController.js``, it is a good practice to have the class and the file having the same name

Reply:
Thanks, resolved